api/mesh/v1alpha1/dataplane.proto

```diff
@@ -47,6 +47,15 @@ message Dataplane {
           [ (validate.rules).uint32 = {gte : 1, lte : 65535} ];
     }
 
+    // Gateway describes a service that ingress should not be proxied ex. API
+    // Gateways
+    message Gateway {
+      // Tags associated with an application this dataplane is deployed next to,
```

Comment:
I would suggest to make it more clear:
`Tags associated with an application this dataplane is deployed next to` => `Tags associated with a gateway (e.g., Kong, Contour, etc) this dataplane is deployed next to`